packages/components-vue/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { createApp } from 'vue';
+import App from './App.vue';
+import router from './router';
+import { componentsReady } from '@porsche-design-system/components-vue';
+
+const app = createApp(App);
+
+app.use(router);
```

Comment:
So I guess even though other libraries go the plugin way, we don't provide any plugin like stuff here and wait for consumer feedback?

Reply:
I will discuss it with Alvaro and get some feedback beforehand